app/assets/stylesheets/components/_blue-box-campaign.scss

```diff
@@ -0,0 +1,45 @@
+$light-grey: #bfc1c3;
+$blue: #005ea5;
+$white: #ffffff;
+
+.app-c-native-blue-box-campaign__border {
+  display: block;
+  border-top: 1px solid $light-grey;
+  padding: 20px 0 0;
+  text-align: center;
+}
+
+.app-c-native-blue-box-campaign__link {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: $blue;
+  width: 100%;
+  min-height: 280px;
+  color: $white;
+  text-decoration: none;
+  text-align: left;
+  box-sizing: border-box;
+  padding: 15px;
+  font-size: 27px;
+  line-height: 30px;
+  font-weight: bold;
+}
+
+.app-c-native-blue-box-campaign__title {
+  display: block;
+  color: $white;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+.app-c-native-blue-box-campaign__description {
+  display: block;
+  font-weight: normal;
+  padding-top: 10px;
+  font-size: 19px;
```

Comment:
As above: I would suggest trying to use GOV.UK Elements font mixins, e.g: @include core-19 in this case. I think this includes the line-height.